### PR TITLE
Prevent right border of cell from moving.

### LINKED ISSUE #645: Marking cell shift cell position horizontally...
By 1 pixel, because the border-left is wider. 

Not "Critical" for 4.1. so marking 4.x, as once you see it you cannot un see it.

### PR DESCRIPTION
**Pull request by jdfreder**
closes #645

### CODE REVIEW

**Comment by jdfreder** [comment]
Action shot:

bugfix

@Carreau @ellisonbg

File: notebook/static/notebook/less/cell.less [hunk 1]
```diff
@@ -27,6 +27,7 @@ div.cell {
         border-left-color: @marked_border_color;
         border-left-width: 3px;
         margin-left: -2px;
+        width: ~"calc(100% + 2px)";
```
Comment (Carreau) [inline notebook/static/notebook/less/cell.less]: meh, I don't trust calc:

```
div.cell.marked {
    border-left-color: #009af5;
    border-left-width: 3px;
    padding-left: 3px; /**replace the negative margin **/
}
```

Comment (Carreau) [inline notebook/static/notebook/less/cell.less]: Though it seem we could use it: http://caniuse.com/#search=calc

Comment (jdfreder) [inline notebook/static/notebook/less/cell.less]: I too prefer the padding fix you posted

Comment (jdfreder) [inline notebook/static/notebook/less/cell.less]: :heavy_check_mark:

**Comment by jdfreder** [comment]
@Carreau last commit uses your padding fix

**Comment by Carreau** [comment]
Haha you couldn't `this.merge()`

**Pull request merged**